k8s节点notReady时，pod和pvc自动迁移失败

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
-->


**What happened**:
k8s节点notReady时，pod和pvc自动迁移失败

**What you expected to happen**:
k8s节点notReady时，pod和pvc自动迁移成功

**How to reproduce it**:
1. 创建deploy和pvc，pod可以正常running
2. 查询pod运行的节点
3. 登录pod运行的节点，执行`systemctl stop kubelet`
4. 查看csi-carina-controller日志，等待csi-carina-controller的检查周期， `kubectl logs -f --tail 10 csi-carina-provisioner-d78c4f474-7xs52 -n kube-system -c csi-carina-controller`

**Anything else we need to know?**:
csi-carina-controller日志
```shell
2023-03-01T10:44:26.849+0800    info    controllers/node_controller.go:122      logic volume resource reconcile ...
2023-03-01T10:44:26.849+0800    info    controllers/node_controller.go:333      get node  name: node3 ,type: Ready,status: Unknown
2023-03-01T10:44:26.849+0800    info    controllers/node_controller.go:154      nodeStatus list: map[node1:normal node2:normal node3:abnormal]
2023-03-01T10:44:26.849+0800    info    controllers/node_controller.go:186      lv list: pvc-675859dc-8886-41d8-8bec-c48a8d1864b5
2023-03-01T10:44:26.849+0800    info    controllers/node_controller.go:195      checkout lv: pvc-675859dc-8886-41d8-8bec-c48a8d1864b5
2023-03-01T10:44:26.849+0800    info    controllers/node_controller.go:201      start clear pod: node3
2023-03-01T10:44:26.849+0800    info    controllers/node_controller.go:302      not ready node: node3  pod: test-carina-lvm-xfs-recover-77b5d5877d-kndc5 
2023-03-01T10:44:26.909+0800    info    controllers/node_controller.go:65       pod test-carina-lvm-xfs-recover-77b5d5877d-kndc5 reconcile manager...
2023-03-01T10:44:26.909+0800    info    controllers/node_controller.go:122      logic volume resource reconcile ...
2023-03-01T10:44:26.909+0800    info    controllers/node_controller.go:333      get node  name: node3 ,type: Ready,status: Unknown
2023-03-01T10:44:26.909+0800    info    controllers/node_controller.go:154      nodeStatus list: map[node1:normal node2:normal node3:abnormal]
2023-03-01T10:44:26.909+0800    info    controllers/node_controller.go:186      lv list: pvc-675859dc-8886-41d8-8bec-c48a8d1864b5
2023-03-01T10:44:26.910+0800    info    controllers/node_controller.go:195      checkout lv: pvc-675859dc-8886-41d8-8bec-c48a8d1864b5
2023-03-01T10:44:26.910+0800    info    controllers/node_controller.go:201      start clear pod: node3
2023-03-01T10:44:26.910+0800    info    controllers/node_controller.go:207      Namespace: default Name: pvc-carina-lvm-xfs-recover Status: Success
2023-03-01T10:44:26.913+0800    info    controllers/node_controller.go:348      delete pod namespace: default name: test-carina-lvm-xfs-recover-77b5d5877d-kndc5
2023-03-01T10:44:26.913+0800    info    controllers/node_controller.go:207      Namespace: default Name: pvc-carina-lvm-xfs-recover Status: Success
2023-03-01T10:44:26.919+0800    info    controllers/node_controller.go:186      lv list: pvc-a2b24e34-bc43-4ce1-898d-afeb47674d3b
2023-03-01T10:44:26.919+0800    info    controllers/node_controller.go:195      checkout lv: pvc-a2b24e34-bc43-4ce1-898d-afeb47674d3b
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x152caa9]

goroutine 199 [running]:
github.com/carina-io/carina/controllers.(*NodeReconciler).getNeedRebuildVolume(0xc000540948, {0x1b2afc8, 0xc00013c000})
        /workspace/github.com/carina-io/carina/controllers/node_controller.go:197 +0x809
github.com/carina-io/carina/controllers.(*NodeReconciler).resourceReconcile(0xc000b4ff90, {0x1b2afc8, 0xc00013c000})
        /workspace/github.com/carina-io/carina/controllers/node_controller.go:123 +0x89
github.com/carina-io/carina/controllers.(*NodeReconciler).SetupWithManager.func2(0xc000294320)
        /workspace/github.com/carina-io/carina/controllers/node_controller.go:87 +0xa5
created by github.com/carina-io/carina/controllers.(*NodeReconciler).SetupWithManager
        /workspace/github.com/carina-io/carina/controllers/node_controller.go:82 +0x1be
```

**Environment**:
- CSI Driver version:
- Kubernetes version (use `kubectl version`):
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

--- a/controllers/node_controller.go
+++ b/controllers/node_controller.go
@@ -194,7 +194,11 @@ func (r *NodeReconciler) getNeedRebuildVolume(ctx context.Context) (map[string]c
 
 		log.Infof("checkout lv: %s", lv.Name)
 		if isSkip, err := r.skipLv(ctx, lv); isSkip || err != nil {
-			log.Errorf("skip lv whether the pod.annotation meets the condition in not ready node:%s  err:%s", lv.Spec.NodeName, err.Error())
+			if err == nil {
+				log.Infof("skip lv whether the pod.annotation meets the condition in not ready node:%s", lv.Spec.NodeName)
+			} else {
+				log.Errorf("skip lv whether the pod.annotation meets the condition in not ready node:%s  err:%s", lv.Spec.NodeName, err.Error())
+			}
 			continue
 		}
 

